app/src/main/java/org/openobservatory/ooniprobe/test/TestAsyncTask.java

```diff
@@ -167,25 +160,35 @@ private void downloadURLs() {
                     app, BuildConfig.SOFTWARE_NAME, BuildConfig.VERSION_NAME, new LoggerArray(), proxy));
             OONIContext ooniContext = session.newContextWithTimeout(30);
 
-            ThirdPartyServices.addLogExtra("ooniContext", app.getGson().toJson(ooniContext));
-
-            session.maybeUpdateResources(ooniContext);
-            OONIURLListConfig config = new OONIURLListConfig();
-            config.setCategories(app.getPreferenceManager().getEnabledCategoryArr().toArray(new String[0]));
+            /* TODO(bassosimone,aanorbel): figure out how to initialize onWifi
+                and charging by inspecting ServiceUtil.java
+             */
```

Comment:
@aanorbel what do you think is the best way to do this? (Ideally we'd like to factor some common code?)